src/app-logic/url-handling.js

```diff
@@ -100,7 +100,7 @@ type BaseQuery = {|
   hiddenThreads: string, // "0-1"
   profiles: string[],
   profileName: string,
-  showTabOnly: BrowsingContextID,
+  showTabOnly1: BrowsingContextID,
```

Comment:
I'm assuming once we launch, this goes back to `showTabOnly`

Reply:
Yep!